prisma/schema.prisma

```diff
@@ -0,0 +1,207 @@
+generator client {
```

Comment:
Try to use multi file support to better organize the schema ya
https://www.prisma.io/blog/organize-your-prisma-schema-with-multi-file-support